spec/integration/analytics_data_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'AnalyticsDataTest' do
   before do
     allow(GovukIndex::MigratedFormats).to receive(:migrated_formats).and_return({})
-    @analytics_data_fetcher = AnalyticsData.new(SearchConfig.instance.base_uri, %w(government_test govuk_test))
+    @analytics_data_fetcher = AnalyticsData.new("http://localhost:9200", %w(government_test govuk_test))
```

Comment:
Should this be `SearchConfig.default_instance.base_uri` or `Clusters.default_cluster.uri`?